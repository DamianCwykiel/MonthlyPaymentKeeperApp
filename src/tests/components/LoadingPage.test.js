import React from 'react';
import { shallow } from 'enzyme';
import LoadingPage from '../../components/LoadingPage';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

//Adapter configuration
Enzyme.configure({ adapter: new Adapter() });

test('shpuld render loadingPage', () => {
    const wrapper = shallow(<LoadingPage />);
    expect(wrapper).toMatchSnapshot();
});