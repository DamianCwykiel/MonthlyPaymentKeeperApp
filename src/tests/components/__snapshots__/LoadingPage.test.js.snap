// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shpuld render loadingPage 1`] = `""`;
